src/esp/physics/bullet/BulletRigidObject.h

```diff
@@ -257,31 +257,31 @@ class BulletRigidObject : public BulletBase,
    * @return The scalar friction coefficient of the object.
    */
   double getFrictionCoefficient() const override {
-    return bObjectRigidBody_->getFriction();
+    return static_cast<double>(bObjectRigidBody_->getFriction());
```

Comment:
nice.  been wanting to take care of these for a long time, just never got around to it.